packages/tinygo/build.sh

```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_HOMEPAGE=https://tinygo.org
+TERMUX_PKG_DESCRIPTION="Go compiler for small places."
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=0.22.0
+TERMUX_PKG_SRCURL=https://github.com/tinygo-org/tinygo/archive/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=d8456c82682d1ece1285887573eb7c421a47e87ffe59ed76ecd87af6de3ba886
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_DEPENDS="golang,libllvm,cmake,ninja,build-essential,git,python"
```

Comment:
`TERMUX_PKG_DEPENDS` is for run-time dependency. I believe (at least) `cmake`, `ninja` and `build-essential` need not be contained.

Reply:
> `TERMUX_PKG_DEPENDS` is for run-time dependency. I believe (at least) `cmake`, `ninja` and `build-essential` need not be contained.

../../../go/pkg/mod/tinygo.org/x/go-llvm@v0.0.0-20220121152956-4fa2ab2718f3/IRBindings.h:16:10: fatal error: 'llvm-c/Core.h' file not found